xbmc/video/ContextMenus.cpp

```diff
@@ -276,24 +292,44 @@ class CVideoPlayActionProcessor : public VIDEO::GUILIB::CVideoPlayActionProcesso
 
   bool OnPlaySelected() override
   {
-    Play();
+    std::string player;
+    if (m_choosePlayer)
+    {
+      const CPlayerCoreFactory& playerCoreFactory{CServiceBroker::GetPlayerCoreFactory()};
+      const std::vector<std::string> players{GetPlayers(playerCoreFactory, *m_item)};
+      player = playerCoreFactory.SelectPlayerDialog(players);
+      if (player.empty())
+      {
+        m_userCancelled = true;
```

Comment:
sorry, haven't fully understood the action processor yet (hard to read on Github), but wouldn't you have to reset ` m_choosePlayer`  here as well and not just ini the ` Play()`  method?

Reply:
Sorry, no.